### PR TITLE
Update 2021-08-10-poly-network.md

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2021-08-10-poly-network.md [hunk 1]
```diff
@@ -1,26 +1,37 @@
 ---
 date: 2021-08-10
-categories: DeFi
-title: Poly Network Exploited
+custodians: Poly Network
+categories: Protocol exploit
+title: Poly Network Hack - $610 Million Stolen in 2021
```
Comment (jhirschkorn) [inline content/attacks/posts/2021-08-10-poly-network.md:5]: Add quotes to title

File: content/attacks/posts/2021-08-10-poly-network.md [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
+- **August 10, 2021:** [Poly Network is hacked.](https://twitter.com/PolyNetwork2/status/1425073987164381196)
+- **August 10, 2021:** [Tether froze ~33M $USDt on 0xC8a65Fadf0e0dDAf421F28FEAb69Bf6E2E589963](https://twitter.com/paoloardoino/status/1425090760609832978)
+- **August 12, 2021:** [The attackers return most of the stolen funds](https://twitter.com/PolyNetwork2/status/1425733950614360064)
+- **August 23, 2021:** [Poly Network Commences Full Asset Restoration](https://medium.com/poly-network/poly-network-commences-full-asset-restoration-7f5c548423b9)
 
 ## Security Failure Causes
 
-- Lack of security audits and vulnerability assessments.
+- **Vulnerabilities in Poly Network's code:** The attackers exploited a vulnerability in Poly Network's code to transfer the funds to their own wallets.
+- **Lack of security updates:** Poly Network had not released a sec
```
Comment (jhirschkorn) [inline content/attacks/posts/2021-08-10-poly-network.md:37]: ?

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Review by jhirschkorn — APPROVED** [review]

**Comment by albina-at-inca** [comment]
@kishoian please send me your preferred payment method along with the wallet address to wiki.challenge.bounties@dn.institute

**Pull request merged**